datatypes/src/operations/image/colorizer.rs

```diff
@@ -18,19 +18,25 @@ pub enum Colorizer {
     LinearGradient {
         breakpoints: Breakpoints,
         no_data_color: RgbaColor,
-        default_color: RgbaColor,
+        default_color: RgbaColor, // TODO: deprecated, this will be removed. Please use over/under values.
+        over_color: Option<RgbaColor>,
+        under_color: Option<RgbaColor>,
```

Comment:
It should be the other way around.
Maybe you do something like an "either"… either `default_color` or `over_color`.

But you serialize only into `over_color` and `under_color`, not `default_color` anymore.

And maybe you can deserialize this either type directly into this pair of colors :thinking: .

Reply:
Ok, I really hope that I'm not misunderstanding something here, but this is the new implementation:
https://github.com/geo-engine/geoengine/pull/684/commits/1667bc7a42482d61c00da0265b0de578c593e20f#diff-36b1b82e93e26eceb563391b9c7a3994a3254b9416494a2d64d3b3bfd5da45b1R23